tests/session/session-isInTransaction-001.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\Session::isInTransaction()
```

Comment:
I think this test would do well to cover both the "started" and "in progress" transaction states. At the moment, I believe it only covers "started", which is before the first command is sent. See [`test_in_transaction()`](https://github.com/mongodb/mongo-c-driver/commit/8868a3bbe79fe9ca826b64fcb7df17c19fbb5bb7#diff-c96a6ebf9cd16bc38cf93c1d65dc69e8R112) for a good example. Ideally, we should just port that to a PHP test.

Reply:
Our test was basically that test already, but I added two sections that actually have an operation in the transaction too.